packages/tools/client-debugger/client-debugger-view/src/components/AudienceView.tsx

```diff
@@ -87,158 +86,92 @@ export function AudienceView(props: AudienceViewProps): React.ReactElement {
 	myClientConnection: IClient | undefined;
 }
 
 /**
- * Displays a list of current audience members and their metadata.
+ * Removes unncessary data in audienceData.audienceState
  */
-function MembersView(props: MembersViewProps): React.ReactElement {
-	const { audience, myClientId, myClientConnection } = props;
-
-	const transformedAudience = combineMembersWithMultipleConnections(audience);
-
-	const memberViews: React.ReactElement[] = [];
-	for (const member of transformedAudience.values()) {
-		memberViews.push(
-			<StackItem key={member.userId}>
-				<AudienceMemberView
-					audienceMember={member}
-					myClientId={myClientId}
-					myClientConnection={myClientConnection}
-				/>
-			</StackItem>,
-		);
-	}
-
-	return <Stack>{memberViews}</Stack>;
+function AudienceStateDataFilter(
+	audienceStateData: AudienceClientMetadata[],
+	myClientConnection: IClient | undefined,
+): FilteredAudienceStateData[] {
+	return audienceStateData.map((entry) => {
+		const clientId = entry.clientId;
+		const userId = entry.client.user.id;
+		const mode = entry.client.mode;
+		const scopes = entry.client.scopes;
+
+		return {
+			clientId,
+			userId,
+			mode,
+			scopes,
+			myClientConnection,
+		};
+	});
 }
 
 /**
- * {@link HistoryView} input props.
+ * Filtered audience state data for {@link AudienceHistoryDataFilter}
  */
-interface HistoryViewProps {
-	/**
-	 * History of audience changes tracked by the debugger.
-	 */
-	history: readonly AudienceChangeLogEntry[];
+export interface FilteredAudienceHistoryData {
+	clientId: string;
+	time: string;
+	changeKind: string;
 }
 
 /**
- * Displays a historical log of audience member changes.
+ * Removes unncessary data in audienceData.audienceHistory
  */
-function HistoryView(props: HistoryViewProps): React.ReactElement {
-	const { history } = props;
-
+function AudienceHistoryDataFilter(
+	audienceHistoryData: readonly AudienceChangeLogEntry[],
+): FilteredAudienceHistoryData[] {
 	const nowTimeStamp = new Date();
-	const historyViews: React.ReactElement[] = [];
 
-	// Reverse history such that newest events are displayed first
-	for (let i = history.length - 1; i >= 0; i--) {
-		const changeTimeStamp = new Date(history[i].timestamp);
+	return audienceHistoryData.map((entry) => {
+		const changeTimeStamp = new Date(entry.timestamp);
 		const wasChangeToday = nowTimeStamp.getDate() === changeTimeStamp.getDate();
 
-		const accordianBackgroundColor: IStackItemStyles = {
-			root: {
-				background: history[i].changeKind === "added" ? "#90ee90" : "#FF7377",
-				borderStyle: "solid",
-				borderWidth: 1,
-				borderColor: DefaultPalette.neutralTertiary,
-				padding: 3,
-			},
-		};
+		const clientId = entry.clientId;
+		const time = wasChangeToday
+			? changeTimeStamp.toTimeString()
+			: changeTimeStamp.toDateString();
+		const changeKind = entry.changeKind;
 
-		const iconStyle: IStackItemStyles = {
-			root: {
-				padding: 10,
-			},
+		return {
+			clientId,
+			time,
+			changeKind,
 		};
```

Comment:
Same here (will need prettier run on it):
```suggestion
		return {
			clientId: entry.clientId,
			time: wasChangeToday
			? changeTimeStamp.toTimeString()
			: changeTimeStamp.toDateString(),
			changeKind: entry.changeKind,
		};
```

Reply:
Resolved.